tests/index.html

```diff
@@ -22,12 +21,10 @@
     {{content-for "body"}}
     {{content-for "test-body"}}
 
-    <script src="{{rootURL}}testem.js" integrity=""></script>
+    <script src="/testem.js" integrity=""></script>
```

Comment:
Shouldn't we use `{{rootURL}}` here to match the rest? Did this change come from Ember?